src/styles/ProjectSummary.scss

```diff
@@ -7,6 +7,11 @@
 
 @media screen and (min-width: $sm) {
     .ProjectSummary {
-
+        .redirect {
```

Comment:
These styles are pretty generic. If there is really nothing built into Material UI that adds a cursor pointer and changes to grey on hover we should at least make a `grey-link` class in a global style that can be re-used